psi4/src/export_functional.cc

```diff
@@ -111,6 +112,9 @@ void export_functional(py::module &m) {
         .def("set_grac_shift", &SuperFunctional::set_grac_shift, "Sets the GRAC bulk shift value.")
         .def("set_grac_alpha", &SuperFunctional::set_grac_alpha, "Sets the GRAC alpha parameter.")
         .def("set_grac_beta", &SuperFunctional::set_grac_beta, "Sets the GRAC beta parameter.")
+        .def("set_density_tolerance", &SuperFunctional::set_density_tolerance, "Sets the density threshold for LibXC.")
+        .def("copy_density_tolerance", &SuperFunctional::copy_density_tolerance, "Copy the density threshold variable to the SuperFunctional layer.")
+        .def("print_LibXC_dens_threshold", &SuperFunctional::py_print_LibXC_dens_threshold, "Querries the LibXCFunctionals for their density threshold values")
```

Comment:
`_libxc_` should be lowercase and dens -> density. This particular function does start to have a leaky abstraction, is there any way this could be `print_density_threshold`?

Reply:
What does leaky abstraction mean? I can rename this, yes. I have no strong opinion about the naming of any functions.

Reply:
The idea here is to tuck all of the detail of LibXC behind the Functional/SuperFunctional classes so if we need to add another library it is easy to do. Here SuperFunctional now mentions LibXC when the practicalities should be hidden.